### PR TITLE
This adds a configuration section to the docs

### PR DESCRIPTION
**Pull request by joncodo**

### CODE REVIEW

**Comment by joncodo** [comment]
@seanspeaks I am not sure why all these commits are here since I just edited this in the github ui. Maybe you can help shed some light here. I really just want my one commit merged.

**Comment by seanspeaks** [comment]
@joncodo Is this supposed to be PR'ed into `gitbook-updates`?

I just brought `main` and `gitbook-updates` relatively up to speed with each other, then merged in your other PRs into gitbook-updates. I figure once this is sorted out, we'd then just true up gitbook-updates... not a great flow, currently forced on us by GitBook because they haven't resolved how to make PRs/merges into an authenticated branch. (used to be direct to main from GitBook and vice versa, but once you lock down a branch, you need a separate branch, and that leads us to getting stale until/unless we automate syncing those branches OR GitBook solves the override.

File: docs/develop/tutorials/quick-start/create-frigg-app.md [hunk 1]
```diff
@@ -8,14 +8,33 @@ Use `npx` to use the latest generator code, and just pass it whatever you want t
...
 npx create-frigg-app [my-app-integrations]
+cd [my-app-integrations]
 ```
 
 Congrats! You've just successfully scaffolded and installed your Frigg application.
 
+## Configuration
+
+### Frontend
+You will have to set up your own config files for the frontend of your applicaiton with:
```
Comment (seanspeaks) [inline docs/develop/tutorials/quick-start/create-frigg-app.md:19]: application* (typo)

**Comment by joncodo** [comment]
Closing in favour of https://github.com/friggframework/frigg/pull/88

**Pull request closed without merging**